docs/tutorials/4_train_airl.ipynb

```diff
@@ -92,51 +88,52 @@
     "from imitation.algorithms.adversarial.airl import AIRL\n",
     "from imitation.rewards.reward_nets import BasicShapedRewardNet\n",
     "from imitation.util.networks import RunningNorm\n",
-    "from imitation.util.util import make_vec_env\n",
     "from stable_baselines3 import PPO\n",
+    "from stable_baselines3.ppo import MlpPolicy\n",
     "from stable_baselines3.common.evaluation import evaluate_policy\n",
     "\n",
-    "import gym\n",
-    "import seals\n",
-    "\n",
     "\n",
-    "venv = make_vec_env(\"seals/CartPole-v0\", n_envs=8, rng=rng)\n",
+    "env = make_vec_env(\"seals/CartPole-v0\", n_envs=8, rng=np.random.default_rng(SEED))\n",
```

Comment:
Why are we re-creating the environment rather than using existing `env`?

Reply:
fixed